packages/danger/src/ChangelogChecker.ts

```diff
@@ -65,7 +65,8 @@ async function runAddChangelogCommandAsync(
   suggestedEntries: PackageChangelogEntry[]
 ): Promise<FixedChangelogEntry[]> {
   for (const entry of suggestedEntries) {
-    await spawnAsync('et', [
+    await (path.join(getExpoRepositoryRootDir(), 'bin', 'expotools'),
```

Comment:
Missing `spawnAsync`? 😄

Reply:
Coding and participating in the lecture at the same time is, not always the best idea 😅